contracts/zero-ex/contracts/src/features/TransformERC20.sol

```diff
@@ -220,31 +241,44 @@ contract TransformERC20 is
                 .getSpendableERC20BalanceOf(inputToken, taker);
         }
 
-        IFlashWallet wallet = getTransformWallet();
+        TransformERC20PrivateState memory state;
```

Comment:
Was this to work around a stack too deep error? The general downside to this approach is that we're now declaring w/o initializing, so a developer may accidentally reference a field before it's been populated.

Reply:
Yeah. I don't think there's a way around that problem.